### ISSUE TITLE
Strange Javascript blocking loophole?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#84]
```
What steps will reproduce the problem?
1. At a trusted homepage (eg. google.com) - Obviously javascript worked.
2. From bookmark go to other site which need javascript (eg 
http://www.ajaxdaddy.com/demo-jquery-carousel.html) 
3. Seem the script execute.
4. If then clicked on a link or refresh the page, then all blocked by default 
as usual.

What is the expected output? What do you see instead?

From any ALLOWED/TRUSTED /TEMP site if user go to another new site/pages, if 
there is no user authorization, then the 1st load page should block script by 
default.

Instead, it only blocked when user click any link or refresh the pages.

What version of the product are you using? On what operating system?

Scriptno Version 1.0.6.2
Chrome 18.0.1025.39 Beta
Mac OS X Lion 10.7.3

Please provide any additional information below.

```

Original issue reported on code.google.com by `yeamteik...@gmail.com` on 23 Feb 2012 at 4:47
- Merged into: #66

**Comment by GoogleCodeExporter** [comment]
```
Create a new clean chrome profile and further test.

It seem Scriptno does not block initial loading of a web pages.

For example Google Homepage Black Bar.

It need javascript to display the drop down menu.

Default no whitelist including google.

Open a new tab, go there, the drop down menu work. (Script execute)

Refresh again or click any link, drop down menu does not work anymore (Script 
not execute)
```

Original comment by `yeamteik...@gmail.com` on 24 Feb 2012 at 1:29

**Comment by GoogleCodeExporter** [comment]
```
http://optimalcycling.com/other-projects/notscripts/

Another example.

Inside got a test for those annoying javascript popup.

As usual nothings should happen if we click on that area.

Instead the javascript execution is like business as usual.

As usual nothings is blocked on first loading.

eg, entering from address bar or from bookmark bar.

The only way it blocked is we refresh the pages or right click (or any 
alternative way of opening pages) from bookmark bar.

Scriptno 1.0.5 does not have this problem.

Note: Default option. Blank Whitelist.
```

Original comment by `yeamteik...@gmail.com` on 3 Mar 2012 at 1:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 1 Jan 2013 at 8:22
- Changed state: **Duplicate**
